From 64272082d6727bfef135bb5975aeff5c1a9b8ec9 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:13:00 -0500
Subject: [ethernet] be2net: remove return statements for void functions

Message-id: <1416413587-12091-13-git-send-email-ivecera@redhat.com>
Patchwork-id: 100334
O-Subject: [RHEL7.1 PATCH 12/19] be2net: remove return statements for void functions
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit 627cd5f8a74a2239ac3ca742b710025dee85db1d
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Fri Sep 19 15:46:55 2014 +0530

    be2net: remove return statements for void functions

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_ethtool.c b/drivers/net/ethernet/emulex/benet/be_ethtool.c
index da55e83..0ae3e0f 100644
--- a/drivers/net/ethernet/emulex/benet/be_ethtool.c
+++ b/drivers/net/ethernet/emulex/benet/be_ethtool.c
@@ -975,8 +975,6 @@ static void be_set_msg_level(struct net_device *netdev, u32 level)
       FW_LOG_LEVEL_DEFAULT :
       FW_LOG_LEVEL_FATAL);
  adapter->msg_enable = level;
-
- return;
 }
 
 static u64 be_get_rss_hash_opts(struct be_adapter *adapter, u64 flow_type)
diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index fb342fd..889f2f9 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -3712,8 +3712,6 @@ static void be_netpoll(struct net_device *netdev)
   be_eq_notify(eqo->adapter, eqo->q.id, false, true, 0);
   napi_schedule(&eqo->napi);
  }
-
- return;
 }
 #endif
 
@@ -4391,7 +4389,6 @@ static void be_add_vxlan_port(struct net_device *netdev, sa_family_t sa_family,
  return;
 err:
  be_disable_vxlan_offloads(adapter);
- return;
 }
 
 static void be_del_vxlan_port(struct net_device *netdev, sa_family_t sa_family,
-- 
1.7.1